.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
nit-picky: looks like you are already taking care of enforcing `yarn` usage in `preinstall` script of package.json. In such case `.npmrc` is not necessary as nobody will be able to install using `npm`.

Reply:
good catch - removed file